packages/nx/src/plugins/js/index.ts

```diff
@@ -98,7 +100,9 @@ export const createDependencies: CreateDependencies = (
     const lockFileContents =
       packageManager !== 'bun'
         ? readFileSync(lockFilePath).toString()
-        : execSync(`bun ${lockFilePath}`).toString();
+        : execSync(`bun ${lockFilePath}`, {
```

Comment:
Tested using the following code snip https://gist.github.com/Jordan-Hall/fb5f5d1e09ebc5d90c67cf1d745e05c8. 

This worked using NX own package.json and bun.lockb file. Happy to add test but not sure where